conbench/tests/api/test_compare.py

```diff
@@ -161,6 +166,25 @@ def test_multiple_contexts_for_same_cases_but_they_kinda_line_up(self):
         pairs = CompareRunsAPI._join_results(baselines, contenders)
         assert self._parse_ids_from_pairs(pairs) == {("id1", "id3"), ("id2", None)}
 
+    def test_contender_missing_commit(self):
```

Comment:
This test would fail on `main` with the same issue I was seeing on a Conbench deployment.

Reply:
Thanks for that. Really.